mesh/pmesh.cpp

```diff
@@ -4813,6 +4813,7 @@ void ParMesh::PrintAsOne(std::ostream &out) const
       ParGridFunction *pnodes = dynamic_cast<ParGridFunction *>(Nodes);
       if (pnodes)
       {
+         pnodes->HostReadWrite();
          pnodes->SaveAsOne(out);
```

Comment:
The `HostReadWrite` call seems unnecessary -- `SaveAsOne` already does `HostRead` which should be sufficient.

Reply:
`done`